CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Annotation `alpha.giantswarm.io/aws-metadata-v2` to enable AWS Metadata API v2
+- Annotation `alpha.aws.giantswarm.io/update-max-batch-size` to configure max batch size in ASG update policy on cluster or machine deployment CR.
+- Annotation `alpha.aws.giantswarm.io/update-pause-time` to configure pause between batches in ASG update on cluster or machine deployment CR.
```

Comment:
Just saying, the metadata annotation is not in line with the structure of the other ones.

Reply:
Yeah saw that too, not very happy about that

Reply:
@paurosello  What do you think? how do we want to align them?

Reply:
I think it's legit to put **aws** in there as it's specific to AWS, I will align

Reply:
ack